server/src/jobs/protocols.cpp

```diff
@@ -34,87 +34,112 @@ namespace fty::job {
 enum class Type
 {
     Powercom = 1,
-    Xml      = 2,
-    Snmp     = 3,
+    Snmp     = 2,
```

Comment:
I think xml must be prioritized than snmp

Reply:
there is no (more) priority between protocols... we only check if each protocol is reachable on its default port...